server/etcdserver/server_test.go

```diff
@@ -99,8 +99,12 @@ func TestApplyRepeat(t *testing.T) {
 		SyncTicker:   &time.Ticker{},
 		consistIndex: cindex.NewFakeConsistentIndex(0),
 		uberApply:    uberApplierMock{},
+		kv:           mvcc.New(zaptest.NewLogger(t), be, &lease.FakeLessor{}, mvcc.StoreConfig{}),
```

Comment:
Set `kv` to avoid a nil pointer panic:

https://github.com/etcd-io/etcd/blob/4bb9392289623e8bf05916281bd696fbbe42fd4d/server/etcdserver/server.go#L2132-L2149